link/kprobe.go

```diff
@@ -191,6 +198,20 @@ func kprobe(symbol string, prog *ebpf.Program, opts *KprobeOptions, ret bool) (*
 	}
 
 	if opts != nil {
+		if opts.RetprobeMaxActive < 0 {
+			return nil, fmt.Errorf("requires opts.RetprobeMaxActive >= 0: %w", errInvalidInput)
+		} else if opts.RetprobeMaxActive > 0 {
```

Comment:
Nit: drop the unnecessary else.

```suggestion
		}
		
		if opts.RetprobeMaxActive > 0 {
```